locale/en/knowledge/javascript-conventions/what-is-json.md

```diff
@@ -67,38 +73,42 @@ format. There are several gotchas that can produce invalid JSON as well.
 
 These are all examples of valid JSON.
 
-    {"name":"John Doe","age":32,"title":"Vice President of JavaScript"}
+```js
```

Comment:
What about changing to 'json' instead of 'js' here? Because this is a pure 'json' formation :)